src/main/java/seedu/address/model/task/StartTime.java

```diff
@@ -57,4 +59,9 @@ public boolean equals(Object other) {
     public int hashCode() {
         return value.hashCode();
     }
+
```

Comment:
Should the toString() method return value (which is already a string denoting the time)? Instead of having to create a LocalTime variable and switching back into a String.